examples/plot_visualize_bad_epochs.py

```diff
@@ -146,16 +153,13 @@
 # interpolated are in blue. Bad sensors which are not interpolated are in red.
 # Bad trials are also in red.
 
-from autoreject import plot_epochs  # noqa
-plot_epochs(this_epoch, bad_epochs_idx=ar.bad_epochs_idx,
-            fix_log=ar.fix_log, scalings=dict(eeg=40e-6),
-            title='')
+scalings = dict(eeg=40e-6)
+reject_log.plot_epochs(this_epoch, scalings=scalings)
 
 ###############################################################################
 # ... and the epochs after cleaning with autoreject
 
-epochs_ar.plot(scalings=dict(eeg=40e-6))
-
+epochs_ar.plot(scalings=scalings)
```

Comment:
personally I have struggled with this scalings parameter in MNE. It would be nice to have scalings='auto' or something like that without having to set it individually for each channel type.